### ISSUE TITLE
(1.16.4) Issues with the config

### ISSUE THREAD

**Issue by footwanterfin** [Alex-the-666/Ice_and_Fire#3381]
The biome config doesn't work properly. After I edited it I made a test world which hanged immediatly, after that I made another test world where I immediatly found a mausoleum in a desert even though that's not where I configured them to spawn, and after that the world hanged.
The log:
[latest.log](https://github.com/Alex-the-666/Ice_and_Fire/files/5765057/latest.log)

The config file:
[iceandfire-biomes.txt](https://github.com/Alex-the-666/Ice_and_Fire/files/5765070/iceandfire-biomes.txt)

Edit: Made a third test world, cockatrice were spawning everywhere and I found an out of place mausoleum and a lightning dragon nest, then the world hung.

**Comment by TheBv** [comment]
I created a draft that fixes the issue but it'll take some time until it'll get released.
If I properly grasped what caused the issue you should be able to just add a special character wherever you only used specific biomes. eg.
`mausoleumBiomes = ["minecraft:snowy_taiga", "minecraft:snowy_taiga_hills", "minecraft:snowy_tundra"] ->   mausoleumBiomes = ["&#overworld,minecraft:snowy_taiga", "minecraft:snowy_taiga_hills", "minecraft:snowy_tundra"]`
or any other useage of &,!,#,+ or @.
Basically what would happen if there was a property defined as  ["minecraft:snowy_taiga", "minecraft:snowy_taiga_hills"] it would allow the structure or entity to spawn in any biome.

So if you still want to use your custom config and you don't want to wait until the next version of the mod comes out just add some special character somewhere and it should work!
